conf/login_darkstar.conf

```diff
@@ -32,6 +32,17 @@ stdout_with_ansisequence: no
 console_silent: 0
 
 #--------------------------------
+#TCP/IP parameters
+#--------------------------------
+
+uiLobbyDataIp: 0.0.0.0

```

Comment:
please default to 127.0.0.1 for IP

ui us? typical DSP user isn't going to know what your abbreviation is unless you explain it in comment above.